src/apiMocks/mockData.ts

```diff
@@ -112,11 +112,14 @@ export function createRandomBreach(
 
   faker.seed(options.fakerSeed);
   const isResolved = options.isResolved ?? faker.datatype.boolean();
+  const dataClassesEffected = options.dataClassesEffected ?? [];
+  const resolvedDataClasses = isResolved ? dataClasses : [];
+
   return {
     addedDate: options.addedDate ?? faker.date.recent(),
     breachDate: faker.date.recent(),
     dataClasses: dataClasses,
-    resolvedDataClasses: isResolved ? dataClasses : [],
+    resolvedDataClasses: resolvedDataClasses,
```

Comment:
```suggestion
    resolvedDataClasses,
```

Reply:
Removed redundant key/value pairs in [dd3200d](https://github.com/mozilla/blurts-server/pull/4212/commits/dd3200d38555c0993d42adb94d364cff3670b585).